ext/pdo/pdo_dbh.stub.php

```diff
@@ -388,6 +388,13 @@ class PDO
 
     public function __construct(string $dsn, ?string $username = null, #[\SensitiveParameter] ?string $password = null, ?array $options = null) {}
 
+    public static function connect(
+        string $dsn,
+        ?string $username = null,
+        ?string $password = null,
```

Comment:
Might fix some of the tests
```suggestion
        #[\SensitiveParameter]
        ?string $password = null,
```

Reply:
Do you mean php-src/ext/pdo/tests/sensitive_parameter.phpt? If so, I think that might not be my fault, as I haven't touched that test.

But yes, that probably should be there, and a test for it.